Sources/AblyAssetTrackingCore/Configuration/ConnectionConfiguration.swift

```diff
@@ -62,15 +50,6 @@ public class ConnectionConfiguration: NSObject {
                   authCallback: authCallback)
     }
     
-    @objc
-    public convenience init(clientId: String? = nil, objcAuthCallback: @escaping ObjCAuthCallback) {
-        self.init(
-            apiKey: nil,
-            clientId: clientId,
-            objcAuthCallback: objcAuthCallback
-        )
-    }
-    
     @objc
```

Comment:
Can we remove this `@objc` too?

Reply:
b40d1fa2472a89845d37775e13630befcd0a4118